examples/jsm/animation/AnimationClipCreator.js

```diff
@@ -30,7 +30,7 @@ class AnimationClipCreator {
 
 		const track = new NumberKeyframeTrack( trackName, times, values );
 
-		return new AnimationClip( null, period, [ track ] );
+		return new AnimationClip( undefined, period, [ track ] );
```

Comment:
There are many chances at once it this PR that need  a closer review. 

Do you think you can split the PR so this one only changes JSDoc, not the code?